app/assets/stylesheets/base.scss

```diff
@@ -0,0 +1,61 @@
+a {
+  &.disabled {
+    color: $gray-ccc;
+    cursor: not-allowed;
+    text-decoration: none;
+  }
+}
+
+body {
+  font-family: 'Source Sans Pro', 'Arial Unicode MS', Helvetica, sans-serif;
+}
+
+a, a:hover, a:focus {
+  text-decoration: none;
+  border-bottom: 1px dotted $sul-link-color-border;
+}
+
+h1 {
+  font-family: 'dejavu_sansextralight';
```

Comment:
Properties should be ordered color, font-family, letter-spacing, padding<br>Prefer double-quoted strings